lira/views/lira.xml

```diff
@@ -0,0 +1,272 @@
+        <script type="text/javascript" src="/web/static/src/js/views/calendar/calendar_view.js"></script>
+
+        <script type="text/javascript" src="/web/static/src/js/report/qwebactionmanager.js"/>
+        <script type="text/javascript" src="/web/static/src/js/report/utils.js"/>
+        <script type="text/javascript" src="/web/static/src/js/report/client_action.js"/>
+
+        <script type="text/javascript" src="/web_enterprise/static/src/js/apps.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/app_switcher.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/control_panel.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/menu_mobile.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/view_manager.js"></script>
+
+        <script type="text/javascript" src="/web_enterprise/static/src/js/widgets/debug_manager.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/widgets/user_menu.js"></script>
+        <script type="text/javascript" src="/web_enterprise/static/src/js/widgets/switch_company_menu.js"></script>
+
+        <script type="text/javascript" src="/bus/static/src/js/bus.js"></script>
+
+    </template>
+
+    <template id="assets" name="lira assets">
+        <script type="text/javascript" src="/lira/static/src/js/main.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/app_cache.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/core.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/mixins.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/data/dataset.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/data/dependency_cache.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/data/data.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/data/data_service.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/data/data_service_factory.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/hash_service.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/storage_service.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/web_client.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/helper_methods.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/dialog.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/config.js"/>
+        <script type="text/javascript" src="/lira/static/src/js/framework/toast.js"/>
+
+
+        <script type="text/javascript" src="/lira/static/src/js/lib/external/materialize.js"/>
```

Comment:
@pnajman-modoolar Those external lib files appear to be missing from the PR.

Reply:
Hi @joaoalf , I'll push new updated version of this module where this issue will be resolved.